src/v2/Components/Artwork/FillwidthItem.tsx

```diff
@@ -190,7 +189,7 @@ export const FillwidthItem = styled(FillwidthItemContainer)<
   }
 `
 
-export default createFragmentContainer(FillwidthItem, {
+export default createFragmentContainer(withSystemContext(FillwidthItem), {
```

Comment:
Oh wow, much simpler than I realized 😍

Reply:
Everything in the `v2` folder should have the `SystemContext` present already, but we only get access to these props if components that need them use the consumer... a modal failing to appear is usually a sign of a missing `mediator` prop.

Reply:
Interesting! So FC's in V2 should be able to use the hook, and always have access to `mediator` that way.

But, class components need to either use the wrapped HOC (ie- `withSystemContext`), _or_ require the 

```ts
<SystemContextConsumer>{({ mediator }) => <FillwidthItem mediator={...} />}</SystemContextConsumer>
```

construction?

I wonder why this hasn't plagued other places where this component gets used (maybe it's a bug there too?).

Reply:
Potentially the mediator is already being passed down via props? I wouldn't be surprised if we run into this in some places-- most auth entrypoints are covered by integrity tests that will confirm the modal is opened as expected.